wa/commands/templates/uiauto/UiAutomation.java

```diff
@@ -0,0 +1,57 @@
+package ${package_name};
+
+import android.app.Activity;
+import android.os.Bundle;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import android.support.test.runner.AndroidJUnit4;
+
+import android.util.Log;
+import android.view.KeyEvent;
+
+// Import the uiautomator libraries
+import android.support.test.uiautomator.UiObject;
+import android.support.test.uiautomator.UiObjectNotFoundException;
+import android.support.test.uiautomator.UiScrollable;
+import android.support.test.uiautomator.UiSelector;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.arm.wa.uiauto.BaseUiAutomation;
+
+@RunWith(AndroidJUnit4.class)
+public class UiAutomation extends BaseUiAutomation {
+
+    protected Bundle parameters;
+
+    public static String TAG = "${name}";
+
+@Before
```

Comment:
Is it normal to have these decorators (sorry if wrong jargon) de-indented like this?

Reply:
No they shouldn't have been, thanks for pointing that out.